src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/SharedTableBuilder.java

```diff
@@ -189,6 +192,7 @@
     private String name;
     private AmazonDynamoDB amazonDynamoDb;
     private MtAmazonDynamoDbContextProvider mtContext;
+    private Optional<String> topLevelContext = empty();
```

Comment:
A shared table client must define a top-level context to be able to create multitenant tables. Creating and deleting multitenant tables can be done only when the context matches the top-level context.